great-idea-website/css/index.css

```diff
@@ -63,4 +63,91 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+
+/* Your code starts here! */
+
+body {
```

Comment:
It would make things a lot easier if You defined some kind of fixed width on the body or container element (for example div that will wrap all elements on the page) and centered it with `margin: 0 auto;` . That way it wil be easier to plan out and position all elements.